### PR TITLE
Update requirements, add web apps

### PR DESCRIPTION
**Pull request by manics**
- Updates requirements
- Adds some web apps with configuration
- Removes use of `hosts: all`

This is intended as a reference for the OME2020 meeting.

Alternatively this could go into another repository

### CODE REVIEW

File: requirements.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 
 - src: ome.postgresql
-  version: 4.0.0
+  version: 5.0.1
 
 - src: ome.omero_server
-  version: 3.0.0
+  version: 3.2.0
```
Comment (manics) [inline requirements.yml]:
```suggestion
  version: 3.3.0
```
As soon as Ansible galaxy updates...

**Comment by sbesson** [comment]
My  2p about the addition of the various apps  https://github.com/ome/omero-deployment-examples#public-user initially restricted the scope of this example to recreate the configuration of the public OMERO user configuration.

From my side,  there is definitely value in maintaining a minimal example playbook deploying a more advanced OMERO ecosystem (web apps, plugins, maybe micro-services) though. If this were to be constructed as a new repository, what would be your thoughts in terms of scope and name?

**Comment by manics** [comment]
If you see microservices being added too how about `ansible-example-omero+addons`?

**Comment by joshmoore** [comment]
`s/+/-/` but otherwise that would work for me. Would certainly give us somewhere to explore the micro services.

**Comment by jburel** [comment]
``omero_addons`` sounds good

**Comment by joshmoore** [comment]
> omero_addons

Now it's almost sounding like a role...

File: requirements.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 - src: ome.omero_server
-  version: 3.0.0
+  version: 3.3.0
 
 - src: ome.omero_web
-  version: 3.0.1
+  version: 3.1.0
 
 - src: ome.omero_user
   version: 0.1.2
```
Comment (manics) [inline requirements.yml]:
```suggestion
  version: 0.2.0
```

**Comment by manics** [comment]
Created a new repo https://github.com/manics/ansible-example-omero-addons

**Pull request closed without merging**